class StormError(Exception):
    pass

class StormNotFoundError(StormError):
    pass
